.azure-pipelines/template-skipvstest.yml

```diff
@@ -0,0 +1,18 @@
+steps:
+- ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+  - script: |
+      set -ex
+      for i in $(git diff HEAD~ | grep '^diff ' | awk '{print$NF}' | sed 's#b/##')
+      do
+        # if code is not in /platform, /files/build/versions and .azure-pipelines, don't skip.
+        echo $i | grep -v "^platform" | grep -v "^files/build/versions" | grep -v "^.azure-pipelines" && echo $i && exit 0
+        # if code is in /platform/vs, don't skip
+        echo $i | grep "^platform/vs" && echo $i && exit 0
+        # if code is in .azure-pipelines/run-test-template.yml, don't skip
+        echo $i | grep "^.azure-pipelines/run-test-template.yml" && echo $i && exit 0
+      done
+      set +x
+      echo "Skip vstest jobs"
+      echo "##vso[task.setvariable variable=SKIP_VSTEST;isOutput=true]YES"
+    name: SetVar
```

Comment:
How about changing a meaningful name? For example, EvaluatePRChange, or any better name.
And change the display name as well.

Reply:
We can add other vars in this step in future.
We can change display name